constraints/src/main/scala/explore/constraints/ConstraintsQueries.scala

```diff
@@ -151,7 +151,7 @@ object ConstraintsQueries {
         .void
     )
 
-  def constraintsSubscription(
+  def ConstraintsSubscription(
```

Comment:
We are treating this like a component, thus the upper casing,  but we can go back.